### ISSUE TITLE
Math formulas cannot be converted well

### ISSUE THREAD

**Issue by Demi1024** [jsvine/notebookjs#49]
Hello, when I use `notebookjs`, I find that can't render math formulas well. 

One is that `katex` is not enough rule for exactly matching math formulas, you can see next pic
![image](https://github.com/jsvine/notebookjs/assets/34363724/4b1a8144-44c0-4fcf-bbcb-d8f0e8278533)

the other is related to referenced `markedjs`. When markedjs parses, it mistakenly converts `_`  to `<em>`, and converts `-`to `<ul><li></li></ul>`.  https://github.com/markedjs/marked/issues/1538
Is there any alternative library for marked.js conversion? 

Hope to get your help❤️

**Comment by jsvine** [comment]
Hi and thank you. Can you provide a notebook that demonstrates the problems you're seeing? That would help to find a fix.

**Comment by Demi1024** [comment]
> Hi and thank you. Can you provide a notebook that demonstrates the problems you're seeing? That would help to find a fix.

Yes, I can provide documents to assist in verifying this issue. 
https://codesandbox.io/s/math-katex-6hkx3p?file=/notebook.ipynb
